r124/u_inref/ydb315.csh

```diff
@@ -0,0 +1,62 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+echo "# Run nosetZcomp^ydb315.m to attempt to compile blktoodeep.m with no '-nowarning' flag (expecting warnings)"
```

Comment:
Do an unsetenv of ydb_compile and gtmcompile before line 14 to ensure these variables are unset. Just in case these env vars are defined at entry into this script.